plugin-server/src/worker/ingestion/event-pipeline/2-emitToBufferStep.ts

```diff
@@ -25,24 +25,16 @@ export async function emitToBufferStep(
     }
 
     if (
-        process.env.DELAY_ALL_EVENTS_FOR_TEAMS === '*' ||
-        process.env.DELAY_ALL_EVENTS_FOR_TEAMS?.split(',').includes(event.team_id.toString())
+        process.env.POE_EMBRACE_JOIN_FOR_TEAMS === '*' ||
```

Comment:
We run these tests twice, once with DELAY_ALL_EVENTS_FOR_TEAMS set. We can probably update that.

Reply:
I'm confused - we'd currently want to run it both ways, no?